test/sass/scss/scss_test.rb

```diff
@@ -3908,6 +3909,44 @@ def test_extra_comma_in_mixin_arglist_error
 SCSS
   end
 
+  def test_extra_comma_in_mixin_arglist_multiline
```

Comment:
I don't think there's a lot of value in testing this in the specifically multiline case. The script parser is pretty newline-agnostic in general.